src/roomba_robot/main.c

```diff
@@ -1,109 +1,10 @@
-#include <kipr/wombat.h>
+#include <stdbool.h>
+#include "kipr/wombat.h"
+#include "tasks/stair_task/stair_task.c"
 
-/* Defining Boolean values */
-#define false 0
-#define true 1
-#define boolean int
-
-/* Pre-defining functions */
-void grappling_position();
-void turn_arm_back();
-void calibration();
-void turn_arm();
-
-/*
-    -> Since KIPR provides functions using snake_case, I'd recommend also switching to snake_case for our functions and variables
-    -> Use boolean as return type for TRUE/FALSE values from now on!
-*/
-
-int main()
-{
+int main(){
+    setuprobot(false, false);
     calibration();
```

Comment:
This might cause issues if both files export the calibration if their files; There are numerous ways to solve this, but one of the would be to create a unified calibration function in the ``main.c``.